railties/lib/rails/generators/rails/benchmark/benchmark_generator.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require "rails/generators/named_base"
+
+module Rails
+  module Generators
+    class BenchmarkGenerator < NamedBase
+      argument :reports, type: :array, default: ["before", "after"]
+      class_option :method, type: :string, enum: %w(ips bm bmbm), default: "ips"
+
+      def generate_layout
+        if benchmark_method == "ips"
+          add_ips_to_gemfile unless ips_installed?
+        end
+
+        template("benchmark.rb.tt", "benchmarks/#{file_name}.rb")
+      end
+
+    private
+      def benchmark_method
+        options["method"]
+      end
+
+      def add_ips_to_gemfile
+        gem_group(:development, :test) do
+          gem(ips_gem_name)
```

Comment:
Think we should just append the group here ala `gem "benchmark-ips", group: %i[ development test ]`. It's not like we're automatically inserting the ips gem in an existing dev/test group.